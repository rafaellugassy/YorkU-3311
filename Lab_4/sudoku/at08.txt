set_number(3, 2, 1)
set_number(2, 2, 2)
set_number(1, 3, 3)
set_number(3, 3, 4)

-- Error: value already exists in row reported first
set_number(3, 2, 4)