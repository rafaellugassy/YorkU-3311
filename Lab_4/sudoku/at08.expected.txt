  Setting up a new grid
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(3,2,1)
  Setting up a new grid
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(2,2,2)
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   0   0
  0   0   0   0
->set_number(1,3,3)
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   1   0
  0   0   0   0
->set_number(3,3,4)
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   1   3
  0   0   0   0
->set_number(3,2,4)
  Error: Number already exists in row
  0   0   0   0
  3   2   0   0
  0   0   1   3
  0   0   0   0
